bin/docker_start.sh

```diff
@@ -37,6 +37,9 @@ if [ -d $fixtures_dir ]; then
     done
 fi
 
+# Create superuser
+/create_superuser.sh admin admin@admin.org admin
+
```

Comment:
I'd make this opt-in via envvars here:

```bash
# specify password by setting DJANGO_SUPERUSER_PASSWORD in the env
# specify username by setting OPENZAAK_SUPERUSER_USERNAME in the env
# specify email by setting OPENZAAK_SUPERUSER_EMAIL in the env

if [ -z "${OPENZAAK_SUPERUSER_USERNAME}" ]; then
    python src/manage.py create_initial_superuser \
        --no-input \
        --username "${OPENZAAK_SUPERUSER_USERNAME}" \
        --email "${OPENZAAK_SUPERUSER_EMAIL:-admin\@admin.org}"
fi
```

These three environment variables can then be documented in the environment variables reference: https://github.com/open-zaak/open-zaak/blob/master/docs/installation/config/env_config.md